modules/ding_reservation/js/ding_reservation_reservability.js

```diff
@@ -8,19 +8,26 @@
 
   Drupal.behaviors.ding_reservation = {
     attach: function(context) {
-      $(".ting-object .reserve-button", context).once('check-reservability', function() {
-        var reserveButton = $(this);
-        var entityId = reserveButton.data("entity-id");
+      var entityIds = [];
+      var selector = '.ting-object .reserve-button.check-reservability';
+      $(selector, context).once('check-reservability', function() {
```

Comment:
If this class is purely for JS reasons then please prefix `check-reservability` with `js-`. With that in mind I think we can also drop the `.reserve-button` class from the selector - perhaps even `.ting-object`.

Reply:
Done